Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -353,6 +353,7 @@ RDKit::SparseIntVect<boost::uint32_t> *MorganFingerprintHelper(
     res = RDKit::MorganFingerprints::getFingerprint(
         mol, static_cast<unsigned int>(radius), invars, froms, useChirality,
         useBondTypes, useCounts, false, bitInfoMap);
+          throw "Value of nbits is Negative"
```

Comment:
This isn't how you throw an exception in C++.
I'd suggest you take a look elsewhere in the RDKit C++ codebase and see how ValueError exceptions are thrown. Hint: there's a function called `throw_value_error()` that you may find useful.